packages/react-components/react-toast/src/components/Toaster.tsx

```diff
@@ -19,27 +20,42 @@ export type ToasterProps = Partial<ToasterOptions>;
 
 export const Toaster: React.FC<ToasterProps> = props => {
   const { offset } = props;
-  const { getToastsToRender, isToastVisible, toasterRef } = useToaster<HTMLDivElement>(props);
+  const { getToastsToRender, isToastVisible } = useToaster<HTMLDivElement>(props);
+  const announceRef = React.useRef<Announce>(() => null);
+
+  const announce = React.useCallback<Announce>((message, options) => {
+    announceRef.current(message, options);
+  }, []);
 
   const styles = useStyles();
 
-  return (
-    <Portal>
-      <div ref={toasterRef}>
-        {getToastsToRender((position, toasts) => {
+  const toastsToRender = getToastsToRender((position, toasts) => {
+    return (
+      <div key={position} style={getPositionStyles(position, offset)} className={mergeClasses(styles.container)}>
+        {toasts.map(toastProps => {
           return (
-            <div key={position} style={getPositionStyles(position, offset)} className={mergeClasses(styles.container)}>
-              {toasts.map(toastProps => {
-                return (
-                  <Toast {...toastProps} key={toastProps.toastId} visible={isToastVisible(toastProps.toastId)}>
-                    {toastProps.content as React.ReactNode}
-                  </Toast>
-                );
-              })}
-            </div>
+            <Toast
+              {...toastProps}
+              announce={announce}
+              key={toastProps.toastId}
+              visible={isToastVisible(toastProps.toastId)}
+            >
+              {toastProps.content as React.ReactNode}
+            </Toast>
           );
         })}
       </div>
-    </Portal>
+    );
+  });
+
+  return (
+    <>
+      <AriaLive announceRef={announceRef} />
```

Comment:
Should it be rendered inside a Portal, too?

Reply:
I dunno, should it?

Reply:
v0 `Dropdown` did it in a Portal, https://github.com/microsoft/fluentui/blob/f376f2871a07ce1f0b376de57968c780cdb13e01/packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx#L1800-L1804

Reply:
I don't mind making that change, but while testing on screen reader I don't see any difference whether it's in the portal or not